sqlmesh/lsp/custom.py

```diff
@@ -0,0 +1,22 @@
+from lsprotocol import types
+import typing as t
+from sqlmesh.utils.pydantic import PydanticModel
+
+ALL_MODELS_FEATURE = "sqlmesh/all_models"
+
+
+class AllModelsRequest(PydanticModel):
+    """
+    Request to get all the models that are in the current project.
+    """
+
+    textDocument: types.TextDocumentIdentifier
```

Comment:
what's up with the naming style? :)

Reply:
This is the default naming style for lsp messages. 

Honestly for me it was just a bit of a coin flip that AI made a decision for when deciding on the 

In TS you do this camel, in Python you do snake, 🤷